packages/calcite-components/src/components/text-area/text-area.e2e.ts

```diff
@@ -150,7 +150,7 @@ describe("calcite-text-area", () => {
     await page.setContent("<calcite-text-area></calcite-text-area>");
 
     const element = await page.find("calcite-text-area");
-    element.setAttribute("max-length", 5);
+    element.setAttribute("max-length", "5");
```

Comment:
Stencil's E2EElement.setAttribute had type `(string, any)=>void`
In Lumina, it is `(string, string)=>void` so that it matches the HTMLElement.setAttribute type.
I want to align us closer with DOM typings so that migrating to Vitest browser mode is easier